src/backends/plonky2/recursion/circuit.rs

```diff
@@ -342,6 +342,7 @@ pub fn common_data_for_recursion<I: InnerCircuit>(
         GateRef::new(plonky2::gates::random_access::RandomAccessGate::new_from_config(&config, 4)),
         GateRef::new(plonky2::gates::random_access::RandomAccessGate::new_from_config(&config, 5)),
         GateRef::new(plonky2::gates::random_access::RandomAccessGate::new_from_config(&config, 6)),
+        GateRef::new(plonky2::gates::exponentiation::ExponentiationGate::new(66)),
```

Comment:
```suggestion
        GateRef::new(plonky2::gates::exponentiation::ExponentiationGate::new_from_config(&config)),
```
please use this constructor so that this still works if we ever change the config